src/ngraph/runtime/cpu/builder/allreduce.cpp

```diff
@@ -13,9 +13,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //*****************************************************************************
-#ifdef NGRAPH_DISTRIBUTED
+#ifdef NGRAPH_DISTRIBUTED_ENABLE
```

Comment:
Maybe it is better to unify the format of `#ifdef NGRAPH_DISTRIBUTED_ENABLE` or `#if defined NGRAPH_DISTRIBUTED_ENABLE`?

Reply:
staying with `#ifdef`